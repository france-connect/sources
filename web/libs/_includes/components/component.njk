{% macro component(name, params = {}) %}
  {% include "components/" + name + ".njk" ignore missing %}
{% endmacro %}