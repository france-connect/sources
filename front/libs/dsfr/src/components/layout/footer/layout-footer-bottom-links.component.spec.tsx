import { render } from '@testing-library/react';

import { LayoutFooterBottomLinksComponent } from './layout-footer-bottom-links.component';

describe('LayoutFooterBottomLinksComponent', () => {
  // given
  const itemsMock = [
    {
      a11y: 'any-a11y-mock-1',
      href: 'any-href-mock-1',
      label: 'any-label-mock-1',
    },
    {
      a11y: 'any-a11y-mock-2',
      href: 'any-href-mock-2',
      label: 'any-label-mock-2',
    },
    {
      a11y: 'any-a11y-mock-3',
      href: 'any-href-mock-3',
      label: 'any-label-mock-3',
    },
  ];

  it('should match the snapshot', () => {
    // when
    const { container } = render(<LayoutFooterBottomLinksComponent items={itemsMock} />);

    // then
    expect(container).toMatchSnapshot();
  });
});
