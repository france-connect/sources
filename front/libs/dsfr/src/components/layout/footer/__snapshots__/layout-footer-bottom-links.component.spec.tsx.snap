// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LayoutFooterBottomLinksComponent should match the snapshot 1`] = `
<div>
  <ul
    class="fr-footer__bottom-list"
  >
    <li
      class="fr-footer__bottom-item"
    >
      <a
        class="fr-footer__bottom-link"
        href="any-href-mock-1"
        title="any-a11y-mock-1"
      >
        any-label-mock-1
      </a>
    </li>
    <li
      class="fr-footer__bottom-item"
    >
      <a
        class="fr-footer__bottom-link"
        href="any-href-mock-2"
        title="any-a11y-mock-2"
      >
        any-label-mock-2
      </a>
    </li>
    <li
      class="fr-footer__bottom-item"
    >
      <a
        class="fr-footer__bottom-link"
        href="any-href-mock-3"
        title="any-a11y-mock-3"
      >
        any-label-mock-3
      </a>
    </li>
  </ul>
</div>
`;
